test/testnet/localnet/config/roles/tendermint-node/templates/genesis.json.jinja2

```diff
@@ -0,0 +1,32 @@
+{
+  "genesis_time": "2019-01-24T08:58:42.728987Z",
+  "chain_id": "test-chain-pS4qfB",
+  "consensus_params": {
+    "block_size": {
+      "max_bytes": "22020096",
+      "max_gas": "-1"
+    },
+    "evidence": {
+      "max_age": "100000"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    }
+  },
+  "validators": [
+    {% for node_id, node in secrets.nodes.items() %}
+        {
+            "address": "{{ node.address }}",
+            "pub_key": {
+                "type": "tendermint/PubKeyEd25519",
+                "value": "{{ node.public_key }}"
+            },
+            "power": "{{ node.power }}",
+            "name": "{{ node_id }}"
+        }{% if not loop.last %},{% endif %}
+    {% endfor %}
+  ],
+  "app_hash": ""
+}
```

Comment:
Now this! Is the first real developers problem that we need to resolve. This is where earlier scripts eventually became unmaintainable. As soon as the genesis.json (or the config.toml for that matter) structure changes, this template is out. BUT, experience tells us that it is not going to get updated.

It was Anton's wise words that I had to understand regarding this: we manage config in the tendermint binary, in a specific package. _That's_ the authentic source for config structure, use it! That's why the `testnet` command came to be, which generates testnet configs (toml, genesis, priv_key, etc) for you so you can edit and use it as you see fit. Let's get rid of this template and see what's missing in the `testnet` command that does not allow us to use that to generate the config. - Then implement it.